packages/flutter/lib/src/rendering/error.dart

```diff
@@ -9,6 +9,8 @@ import 'object.dart';
 
 const double _kMaxWidth = 100000.0;
 const double _kMaxHeight = 100000.0;
+/// This is a simplified estimate to offset the text from the top of the screen.
+const double _statusBarHeight = 24.0;
```

Comment:
I like this solution. Done. 

I assume we can't make it ideal because the RenderBox doesn't know its absolute top position in the screen while painting?